src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding: 10px;
+  gap: $daysGap;
+  width: calc(7*100px + 7*1px);
+
+  &__day {
+    box-sizing: border-box;
+    background-color: #eee;
+    width: $dimensions;
+    height: $dimensions;
+    text-align: center;
+    line-height: $dimensions;
+    border: 1px solid black;
+    transition: transform 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px);
+    }
+  }
+
+  @for $day from 1 through 31 {
+    &__day--#{$day}::before {
+      content: '#{$day}';
+      font-size: 30px;
+      font-family: Arial, Helvetica, sans-serif;
```

Comment:
Inside of loops, set only properties that are changing. In this case content is fine but font styles should be inside of __day